javascript-data-model/order-history.js

```diff
@@ -0,0 +1,68 @@
+var orderHistory = [
+  {
+    number: '114-3941689-8772232',
+    total: '$34.00',
+    numberItems: 1,
+    orderDate: 'August 4, 2020',
+    deliveryDate: 'August 8, 2020',
+    recipient: 'JS Masher',
+    itemDescription: [
+      {
+        item1: 'book, $31.55, Javascript for impatient programmers'
+      }],
+    buyAgain: true,
+    viewItem: true,
+    productReview: true,
+    productSupport: false
+  },
+  {
+    number: '113-9984268-1280257',
+    total: '$44.53',
+    numberItems: 1,
+    orderDate: 'July 19, 2020',
+    deliveryDate: 'July 20, 2020',
+    recipient: 'JS Masher',
+    itemDescription: [
+      {
+        item1: 'book, $41.33, The Timeless Way of Building'
+      }],
+    buyAgain: true,
+    viewItem: true,
+    productReview: true,
+    productSupport: false
+  },
+  {
+    number: '114-2875557-9059409',
+    total: '$17.22',
+    numberItems: 1,
+    orderDate: 'July 4, 2020',
+    deliveryDate: 'July 7, 2020',
+    recipient: 'JS Masher',
+    itemDescription: [
+      {
+        item1: 'adapter, $15.98, Gamecube Controller Adapter'
+      }],
+    buyAgain: true,
+    viewItem: true,
+    productReview: true,
+    productSupport: false
+  },
+  {
+    number: '113-2883177-2648248',
+    total: '$138.93',
+    numberItems: 2,
+    orderDate: 'July 3, 2020',
+    deliveryDate: 'July 5, 2020',
+    recipient: 'JS Masher',
+    itemDescription: [
+      {
+        item1: 'controller, $94.95, Gamecube Controller',
+        item2: 'book, $33.99, The Art of Sql'
```

Comment:
Why the numbered properties? What if the order has 50 items, are we going to have 50 individually named indexes with a number at the end? What data type in JavaScript is good for lists of data that don't need individual names?